### PR TITLE
Updated FindOpenVDB to support pyopenvdb and openvdb_houdini

### PR DESCRIPTION
**Pull request by Idclip**
Signed-off-by: Nick Avramoussis <4256455+Idclip@users.noreply.github.com>

### CODE REVIEW

File: cmake/FindOpenVDB.cmake [hunk 1]
```diff
@@ -224,6 +287,28 @@ list(APPEND _OPENVDB_LIBRARYDIR_SEARCH_DIRS
...
+set(OPENVDB_PYTHON_PATH_SUFFIXES
+  ${CMAKE_INSTALL_LIBDIR}/python
+  ${CMAKE_INSTALL_LIBDIR}/python2.7
+  ${CMAKE_INSTALL_LIBDIR}/python3
+  lib64/python
+  lib64/python2.7
+  lib64/python3
+  lib/python
+  lib/python2.7
+  lib/python3
```
Comment (danrbailey) [inline cmake/FindOpenVDB.cmake:299]: In theory a user could have installed multiple versions of OpenVDB - say one against Python2 and one against Python3. Is it worth providing a mechanism for them to pick the specific paths they want here?

**Comment by Idclip** [comment]
> In general these changes look fine to me - is there any way we could actually test this module through CI? A simple binary application that tried to link against OpenVDB using this mechanism?

Yes absolutely - I can do this as part of this PR if you like. Though this is 1/2 PRs that AX depends on, so it'll block that.

File: cmake/FindOpenVDB.cmake (hunk 1 above)
Comment (Idclip) [reply cmake/FindOpenVDB.cmake:299]: Yeah the variable ` ${OPENVDB_${COMPONENT}_ROOT}` and ` ${OPENVDB_${COMPONENT}_LIBRARYDIR}` should do what you're suggesting here. i.e:
`-DOPENVDB_pyopenvdb_ROOT="/path/to/custom/python/install"`

**Comment by Idclip** [comment]
bump

**Comment by Idclip** [comment]
I will update the changelogs and merge this

**Pull request merged**